.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: swift
+osx_image: xcode9.3
+xcode_project: Static.xcodeproj
+
+script: xcodebuild -scheme "$TRAVIS_XCODE_SCHEME" -sdk "$TRAVIS_XCODE_SDK" -destination "$DESTINATION" test
+
+matrix:
+  include:
+    - xcode_scheme: Static-iOS
+      xcode_sdk: iphonesimulator
+      env:
+        - DESTINATION="OS=10.1,name=iPhone 7 Plus"
+    - xcode_scheme: Static-macOS
```

Comment:
There is no `Static-macOS` scheme, so we should be able to remove this.

Reply:
yep, just removed